.github/workflows/sanity-test.yaml

```diff
@@ -435,9 +527,9 @@ jobs:
           if-no-files-found: error
           retention-days: 14
 
-      - name: Notify failure in teams
-        if: failure()
-        uses: ./.github/actions/teams-message
-        with:
-          msg: "[FAILED] Sanity Tests"
-          teams_url: ${{ secrets.TEAMS_CORSO_CI_WEBHOOK_URL }}
+      #- name: Notify failure in teams
+      #  if: failure()
+      #  uses: ./.github/actions/teams-message
+      #  with:
+      #    msg: "[FAILED] Sanity Tests"
+      #    teams_url: ${{ secrets.TEAMS_CORSO_CI_WEBHOOK_URL }}
```

Comment:
will uncomment once PR is approved and ready to merge

Reply:
feel free to uncomment it now.  Just make sure you tell warn people why they're seeing failure alerts where they appear.